docs/topics/pages.md

```diff
@@ -493,3 +493,18 @@ class EventPage(Page):
 
     objects = EventPageManager()
 ```
+
+### Support for proxy page models
+
+```{versionadded} 5.0```
```

Comment:
ditto here

Reply:
There is a backwards-compatibility issue with these changes, so I assumed it would have to be the next major release: 

Projects that previously registered proxy models as Snippets will need to update any existing `Group` permissions to use 'proxy model specific' permissions where appropriate, in addition to those for the 'concrete parent model'.

Reply:
We most certainly need upgrade considerations for this.

@gasman based on this, we'd probably want to delay unless there will be other breaking changes for the next release